ci.sh

```diff
@@ -51,7 +51,7 @@ else
     # Actual tests
     # expands to 0 != 1 if NO_TEST_REQUIREMENTS is not set, if set the `-0` has no effect
     # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02
-    if [ ${NO_TEST_REQUIREMENTS-0} == 1 ]; then
```

Comment:
IIRC this one in particular doesn't *need* quotes for weird curly braces reasons, but I don't think they can hurt and `no-test-requirements` looks to run correctly, so let's just keep them.